bindata/v3.11.0/kube-apiserver/defaultconfig.yaml

```diff
@@ -32,6 +32,8 @@ apiServerArguments:
   - PersistentLocalVolumes=false # disable local volumes for 4.0, owned by sig-storage/hekumar@redhat.com
   minimal-shutdown-duration:
   - 3s # give SDN some time to converge
+  http2-max-streams-per-connection:
+  - 2000  # recommended is 1000, but we need to mitigate https://github.com/kubernetes/kubernetes/issues/74412
```

Comment:
"2000" to make sure it serializes as a string

Reply:
i bet that was it! good catch